docs/cookbook/docker_logs.md

```diff
@@ -0,0 +1,182 @@
+
+```
+nano pipeline/logstash.config
+```
+
+Step 7: Copy and paste the following content into the logstash.config file:
+
+```
+input {
+  udp {
+    port => 12201
+    codec => "plain"
+  }
+}
+
+output {
+  opensearch {
+    hosts => "https://cloud.axiom.co:443/api/v1/datasets/$DATASET_NAME/elastic"
+    user => "Username"
+    password => "auth-token"
+  }
+}
+```
+
+Let's break down the Logstash configuration file provided:
+
+- `input`: This section defines the input plugin(s) for Logstash, specifying how it should receive data.
+
+- `udp`: This input plugin is used to receive data via the UDP protocol. It listens on a specific port and expects data to be sent in UDP packets.
+
+- `port: 12201`: This line specifies the UDP port number (12201) that Logstash should listen on to receive incoming log data.
+
+- `codec: "plain"`: This line specifies the codec for parsing the incoming log data. In this case, it's set to "plain", indicating that the log data is expected to be in plain text format.
+
+This input configuration tells Logstash to listen for log data arriving via UDP on port 12201, with the expectation that the logs will be in plain text format.Now let's move on to the output configuration:
+
+- `output`: This section defines the output plugin(s) for Logstash, specifying where the processed log data should be sent.
+
+- `opensearch`: This output plugin is used to send the processed log data to an OpenSearch (formerly known as Elasticsearch) cluster.
+
+- `hosts`: "https://cloud.axiom.co:443/api/v1/datasets/$DATASET_NAME/elastic": This line specifies the URL or addresses of the OpenSearch cluster where the log data should be sent. In this example, the logs are being sent to "https://cloud.axiom.co:443/api/v1/datasets/$DATASET_NAME/elastic".
+
+- `user: "Username" and password => "auth-token"`: These lines specify the username and password credentials to authenticate with the OpenSearch cluster. In this case, the username is "Username" and the password is "auth-token".
+
+Step 8: Save and close the logstash.config file (Ctrl + X, Y, Enter).
+
+Step 9: Start the Docker containers using Docker Compose:
+
+```
+docker-compose up -d
+```
+
+At this point, your setup should be complete. The Docker logs from the plop service will be sent to Logstash running in the logstash service, and then forwarded to Axiom using the provided configuration.
+
+Make sure you have the necessary credentials (Axiom username and password) and adjust the Axiom endpoint (hosts value) and dataset path accordingly in the logstash.config file.
+
+You can monitor the logs by accessing Axiom or by checking the Logstash container logs using the command:
+
+```
+docker-compose logs -f logstash
+```
+
+Step 10: Create a file named daemon.json at `/etc/docker` and open it for editing:
+
+```
+sudo nano /etc/docker/daemon.json
+```
+
+Step 11: Copy and paste the following content into the daemon.json file:
+```
+{
```

Comment:
Can you update this to the new configuration, where container names are also sent.